common/app/http/GuardianAuthWithExemptions.scala

```diff
@@ -81,9 +101,19 @@ class GuardianAuthWithExemptions(
       if (doNotAuthenticate(request)) {
         nextFilter(request)
       } else {
-        // TODO: in future PR add a permission check here based on user, likely via a function passed in to GuardianAuthWithExemptions
         AuthAction.authenticateRequest(request) { user =>
-          nextFilter(request)
+          if (permissions.hasPermission(requiredPermission, user.email)) {
+            nextFilter(request)
+          } else {
+//            Future.successful(
```

Comment:
nit: do we still need this comment?

Reply:
I think it gets addressed in https://github.com/guardian/frontend/pull/27092/files

Reply:
yeah, I'm normally against commented CODE (and I'm happy to remove if you feel strongly) but as Max points out it is reinstated in my follow-up PR (linked in PR description)